src/doc_scan_service/session/create/task/requested.text.extraction.config.js

```diff
@@ -11,10 +11,15 @@ class RequestedTextExtractionConfig {
   /**
    * @param {string} manualCheck
    *   Describes the manual fallback behaviour applied to each Task
+   * @param {string} chipData
+   *   Describes the chip data requirement for each Task
    */
-  constructor(manualCheck) {
+  constructor(manualCheck, chipData) {
     Validation.isString(manualCheck, 'manualCheck');
     this.manualCheck = manualCheck;
+
+    Validation.isString(chipData, 'chipData', true);
```

Comment:
Will this break anyone's code who is currently calling this with just manual check? In .NET this is public and so I was going to default it to empty string

Reply:
`true` is the optional flag, so it can be `string` or `undefined` - It will be excluded from the payload by default

Reply:
https://github.com/getyoti/yoti-node-sdk/blob/master/src/yoti_common/validation.js#L17

Reply:
Cool, sounds good 👍